ddl/ddl_worker.go

```diff
@@ -381,13 +380,7 @@ func (w *worker) updateDDLJob(t *meta.Meta, job *model.Job, meetErr bool) error
 }
 
 func (w *worker) deleteRange(ctx context.Context, job *model.Job) error {
```

Comment:
Could we remove this function use `w.delRangeManager.addDelRangeJob(ctx, job)` directly?